<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>671</width>
    <height>462</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>130</y>
     <width>191</width>
     <height>32</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>cost:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="startCostSearchEdit"/>
    </item>
    <item>
     <widget class="QLineEdit" name="finishCostSearchEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>130</y>
     <width>181</width>
     <height>32</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_13">
    <item>
     <widget class="QLabel" name="label_13">
      <property name="text">
       <string>reliability:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="startRealiabilitySearchEdit"/>
    </item>
    <item>
     <widget class="QLineEdit" name="finishRealiabilitySearchEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_3">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>190</y>
     <width>181</width>
     <height>32</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_17">
    <item>
     <widget class="QLabel" name="label_17">
      <property name="text">
       <string>comfort:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="startComfortSearchEdit"/>
    </item>
    <item>
     <widget class="QLineEdit" name="finishComfortSearchEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_4">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>190</y>
     <width>191</width>
     <height>32</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_18">
    <item>
     <widget class="QLabel" name="label_18">
      <property name="text">
       <string>country of origin</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="countrySearchEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_5">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>190</y>
     <width>161</width>
     <height>32</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_19">
    <item>
     <widget class="QLabel" name="label_19">
      <property name="text">
       <string>engine type:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="engineSearchEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_6">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>260</y>
     <width>315</width>
     <height>32</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_20">
    <item>
     <widget class="QLabel" name="label_20">
      <property name="text">
       <string>fuel consumption per 100 km:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="startFuelSearchEdit"/>
    </item>
    <item>
     <widget class="QLineEdit" name="finishFuelSearchEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>360</y>
     <width>191</width>
     <height>71</height>
    </rect>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
   <property name="centerButtons">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>130</y>
     <width>161</width>
     <height>32</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>mark of car:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="markSearchEdit"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
